src/tritonserver.cc

```diff
@@ -50,6 +50,12 @@
 #include "triton/common/triton_json.h"
 #include "tritonserver_apis.h"
 
+#ifndef _WIN32
+#include "opentelemetry/sdk/trace/tracer.h"
+namespace otel_trace_api = opentelemetry::trace;
```

Comment:
@oandreeva-nv in your screenshot examples, why are they labeled "unknown_service"? Is there some tag we can put in the source to have these say "Triton" or something in the collector/UI view?

Reply:
It's due to the jaeger. I was planning to investigate if we can do something with this on our side, or if it is something, that need to be addressed on the collector's side